package.json

```diff
@@ -1,14 +1,11 @@
 {
   "name": "@divriots/dockit-react",
-  "version": "0.0.64",
+  "version": "0.1.0",
```

Comment:
what it is ?

Reply:
@gqio please read and continue in https://github.com/divriots/dockit-react/pull/84#discussion_r678311474

Reply:
ah wait, sorry for confusion

I was originally effected by a glitch with branch's base commit which resulted in version update

and as a result I was trapped into thinking that I should bump the version in the PR itself, but now I'm realising I probably should do it differently, but not sure how, Releases pane in Backlight?

Reply:
No one should touch this version I think.... it's either by hand with manual publish, or automatic with Release or Release Action.

Reply:
I'll remove it, but good to keep in mind we should control it if the end result of this PR will be with new default behavior